lib/parser/objects.ts

```diff
@@ -221,10 +221,16 @@ export class OName extends ObjectBase implements I.IHasDefinitions, I.IHasNameTo
   braceLevel?: number; // braceLevel for the expression
   // Workaround for checking of OWrites in associations. Because of overloading they can not be correctly checked.
   // This avoids false positives
+  constraint = false;
   public inAssociation = false;
   constructor(public parent: ObjectBase | OFile, public nameToken: OLexerToken, public write = false, range?: OIRange) {
     super(parent, range ?? nameToken.range);
   }
+  children: OName[] = [];
+  afterComma = false;
```

Comment:
what does `afterComma` describe? Find a more descriptive name or comment its usage in `elaborateExpressionInstantiations`